### PR TITLE
code refactor

### PR DESCRIPTION
**Pull request by Sherly1001**

### CODE REVIEW

File: commands/admin.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ async def delete_category(guild, author, category_name=config.GAME_CATEGORY):
         response = f"{author.display_name} deleted channel {category_name}"
         print(response)
         await category.delete()
+        return True
```
Comment (nitro2) [inline commands/admin.py:69]: Why return True here? Where is False?
Same to other places

File: game/roles/character.py [hunk 2]
```diff
@@ -21,10 +21,8 @@ def __init__(self, interface, player_id, player_name):
         self.player_name = player_name
         # channel_name MUST BE lowercase!
         valid_channel_name = "".join(c for c in player_name if c not in BANNED_CHARS).lower()
-        valid_channel_name = '-'.join(valid_channel_name.split())
-        if len(valid_channel_name) <= 1:
-            valid_channel_name = f'{valid_channel_name}{player_id}'
-        self.channel_name = f"personal-{valid_channel_name}"
+        valid_channel_name = "-".join(valid_channel_name.split())
+        self.channel_name = f"personal-{valid_channel_name}-{player_id}"
```
Comment (nitro2) [inline game/roles/character.py:25]: Please revert this change. `player_id` only use for short username players.

File: game/roles/guard.py [hunk 3]
```diff
@@ -5,31 +5,22 @@ class Guard(Villager):
...
     def is_yesterday_target(self, target_id):
-        if self.yesterday_target:  # Yesterday player id should not 0
-            if self.yesterday_target == target_id:
-                return True
-            else:
-                return False
-        else:
-            return False
-
+        return self.yesterday_target == target_id
```
Comment (nitro2) [inline game/roles/guard.py:23]: Did you run guard test cases?

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
 discord.py==1.7.3
 python-dotenv
 tzlocal<3.0
-APScheduler==3.0.0
\ No newline at end of file
+APScheduler==3.0.0
```
Comment (nitro2) [inline requirements.txt]: Remove this APScheduler @neihousaigaai  to confirm

**Review by nitro2 — CHANGES_REQUESTED** [review]

File: commands/admin.py (hunk 1 above)
Comment (Sherly1001) [reply commands/admin.py:69]: In error case, return value is None.

File: game/roles/character.py (hunk 2 above)
Comment (Sherly1001) [reply game/roles/character.py:25]: What happends when two or more players has the same nickname, they join in the same channel. :v

File: game/roles/guard.py (hunk 3 above)
Comment (Sherly1001) [reply game/roles/guard.py:23]: Not yet, but I belive it work.

File: game/roles/character.py (hunk 2 above)
Comment (nitro2) [reply game/roles/character.py:25]: Change to `f"{message.author.name }-{message.author.discriminator}"` in do_join

File: commands/admin.py (hunk 1 above)
Comment (nitro2) [reply commands/admin.py:69]: Don't see any profit of return True here. Better keep it unchanged. Otherwise you have to change ConsoleInterface too.

**Review by nitro2 — APPROVED** [review]
